cli/src/cmd_auth.rs

```diff
@@ -512,131 +512,26 @@ impl CmdAuthStatus {
 
 #[cfg(test)]
 mod tests {
-    // use pretty_assertions::assert_eq;
-
-    // use crate::cmd::Command;
-
-    // pub struct TestItem {
-    //     name: String,
-    //     cmd: crate::cmd_auth::SubCommand,
-    //     stdin: String,
-    //     want_out: String,
-    //     want_err: String,
-    // }
-
-    // TODO: Auth is shaky with current docker container CI implementation.
-    // remove ignore tag once tests run against mock API server
-    // #[ignore]
-    // #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
-    // #[serial_test::serial]
-    // async fn test_cmd_auth() {
```

Comment:
can you review this comment before deleting it to see if there's anything useful to extract / learn from it?

Reply:
Removed with https://github.com/oxidecomputer/oxide.rs/pull/727